app/src/main/java/com/celzero/bravedns/database/AppInfo.kt

```diff
@@ -95,4 +97,9 @@ class AppInfo {
         this.screenOffAllowed = screenOffAllowed
         this.backgroundAllowed = backgroundAllowed
     }
+
+    fun hasInternetPermission(packageName: String, packageManager: PackageManager): Boolean {
+        // INTERNET permission if defined, can not be denied so this is safe to use
+        return packageManager.checkPermission(Manifest.permission.INTERNET, packageName) == PackageManager.PERMISSION_GRANTED
```

Comment:
Can we use `this.packageName` instead of passing `packageName` as a parameter?

Reply:
Done